### DISCUSSION TITLE
Migration form caouecs/laravel-lang

### DISCUSSION THREAD

**Discussion by JustChapman** [Laravel-Lang/common#35] [Q&A] [answered]
Hi all - I was wondering if anyone knows of an existing migration guide for users of caouecs/laravel-lang to this project. I just inherited an existing laravel project using the caouecs/laravel-lang package and as of right now, dependencies fail to install. Before diving in, I thought I would check here first.

Thanks!

**Comment by caouecs** [comment, marked as answer]
Hello,
just update caouecs/laravel-lang to laravel-lang/lang in your composer.json
